src/config/config.js

```diff
@@ -10,6 +10,7 @@ const Config = {
   publicUrl,
   infuraKey: process.env.REACT_APP_INFURA_KEY,
   network: process.env.REACT_APP_NETWORK || 'fuse',
+  market: process.env.REACT_APP_MARKET,
```

Comment:
do the default here
|| process.env.REACT_APP_NETWORK==='etoro'

Reply:
Done